docs/Networking.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: basics-network
+id: networking
```

Comment:
Use network here OR update the permalink to use docs/networking.html too. I know -- this id system does not appear to work as intended.